src/datapoint.jl

```diff
@@ -0,0 +1,89 @@
+## New Type: DataPoint
+# Struct for carrying statistic and systematic uncertainty of a DataPoint
+"""
+    DataPoint(x,stat,sys)
+Parametric type that stores a value and its respective statistical and systematic
+uncertanties. The uncertainties can either be numbers for symmetric uncertainties
+or a tuple for upper and lower bounds. It can be created by either providing a
+positional arguments ( e.g. `DataPoint(1.0,0.1,0.2) or
+`DataPoint(1.0,(0.09,0.11),(0.21,0.19))``) or using the keyword arguments `stat`
```

Comment:
Somethings odd here with the quotation marks.
I guess the correct form would be 
```julia
`DataPoint(1.0,0.1,0.2)` or `DataPoint(1.0,(0.09,0.11),(0.21,0.19))`
```